docs/demo/custom-style.md

```diff
@@ -30,13 +30,17 @@ You should expect the above text to appear styled like a yellow sticky note, bec
 A portion of Markdown that is highlighted compared to the rest
 :::
 
+### hightlight-inline
+
+You can highlight ==inline text== as well.
+
 ## Advanced styling 
```

Comment:
Let's move this to `markdown.md`, because we are describing a new extended syntax (`==..==`) more than custom style. In the description, we can link to `[[custom-style]]` informing the user of the "highlight-inline" style.

Reply:
done